modin/pandas/test/test_api.py

```diff
@@ -154,7 +154,17 @@ def test_top_level_api_equality():
             except IndexError:
                 pass
 
-    assert not len(difference), "Extra params found in API: {}".format(difference)
+    try:
+        assert not len(difference), "Extra params found in API: {}".format(difference)
+    except AssertionError:
+        if (
+            len(difference) == 1
+            and difference[0][0] == "read_table"
+            and list(difference[0][1].keys())[0] == "storage_options"
```

Comment:
Why is this needed?

Reply:
pandas added `storage_options` parameter to `read_csv`, but not to `read_table`. Because we are implementing `read_csv` and `read_table` using one function `_make_parser_func` there was inconsistency in signature of the functions (test-api job failed). I asked a question related to adding the parameter to `read_table` in pandas repo and currently waiting for a response.

Reply:
pandas guys said it looks like an omission of one of their commits. They took the question on consideration.

Reply:
Can you link the pandas issue?

Reply:
Yes, sure. [39167](https://github.com/pandas-dev/pandas/issues/39167)

Reply:
Now that we have them split in the API, we can just add the parameter to `read_csv`.

Reply:
Yes, I agree.